### ISSUE TITLE
Axis ranges for autoplot of subpopulation added by gs_pop_add are incorrect

### ISSUE THREAD

**Issue by jacobpwagner** [RGLab/ggcyto#55]
In testing another issue, I noticed that `autoplot` had some strange behavior. With the current trunk branches, this snippet:
```
library(flowCore)
library(flowWorkspace)
library(ggcyto)
data("GvHD")
gs <- GatingSet(GvHD)
gate <- rectangleGate(filterId="rGate", list("FL1-H"=c(0, 2500), "FL2-H"=c(0, 2500)))

gs_pop_add(gs, gate, parent= "root")
recompute(gs)

g <- autoplot(gs[[1]], "rGate")
g
```
produces this plot:
![plot_zoom_png](https://user-images.githubusercontent.com/43548373/70654729-c398a400-1c0b-11ea-8b55-2e709e1ef508.png)

where the axis limits are constrained to 0-1023:
```
> g@.Data[[1]]$coordinates$limits
$x
[1]    0 1023

$y
[1]    0 1023
```
While before the recent `cytoset` changeover (tested using the `bioconductor/devel_cytometry2` docker image), the plot and limits are as we would expect:
![plot_zoom_png](https://user-images.githubusercontent.com/43548373/70654997-4cafdb00-1c0c-11ea-8c65-82d5a9da388d.png)

```
> g@.Data[[1]]$coordinates$limits
$x
[1]     1 10000

$y
[1]     1 10000
```
It's probably just an issue of axis transformation defaults, so it's probably not a very hard fix.

**Comment by jacobpwagner** [comment]
It looks like this is actually arising from the `GatingSet(<flowSet>)` constructor, which calls `flowSet_to_cytoset`, which in turn calls `write.flowSet`. In the process of writing out the `flowSet`, the information about the transformation of the ranges is getting lost: 
```
> library(flowCore)
> data("GvHD")
> fs <- GvHD
> range(fs[[1]])
    FSC-H SSC-H FL1-H FL2-H FL3-H FL2-A FL4-H Time
min     0     0     1     1     1     0     1    0
max  1023  1023 10000 10000 10000  1023 10000 1023
> tmp <- tempfile()
> write.flowSet(fs, tmp, filename = sampleNames(fs))
[1] "/tmp/Rtmpd3iOnh/file2b0949bc27fa"
> fs_read <- read.flowSet(path = tmp, phenoData = "annotation.txt")
> range(fs_read[[1]])
    FSC-H SSC-H FL1-H FL2-H FL3-H FL2-A FL4-H Time
min     0     0     0     0     0     0     0    0
max  1023  1023  1023  1023  1023  1023  1023 1023
```
So it looks like may need to fix that, but this is really probably more of a `flowCore` issue.

**Comment by mikejiang** [comment]
It was malformed `flowFrame` from `GvHD` dataset that has incorrect `PnR` value in the first place, 
```r
> keyword(fr)[["$P3R"]]
[1] "1024"
```
but its maxRange in `fr@parameters@data` slot somehow has the correct value 
```r
> fr@parameters@data
     name              desc range minRange maxRange
$P1 FSC-H        FSC-Height  1024        0     1023
$P2 SSC-H        SSC-Height  1024        0     1023
$P3 FL1-H         CD15 FITC  1024        1    10000
$P4 FL2-H           CD45 PE  1024        1    10000
...
```
and when it was output to FCS by `write.FCS`, only the original (wrong) `PnR` (not `maxRange`)  gets written. and when it is reloaded the `maxRange` is populated by `PnR` automatically.

So I am not sure if what should correct `PnR` keyword based on `maxRange` during `write.FCS`, since normally `PnR` is from the instrument and is supposed to be more accurate.

Maybe we want to correct `maxRange` for `GvHD`?

**Comment by jacobpwagner** [comment]
`GvHD` was fixed by RGLab/flowCore@982bb1b, so I'm going to go ahead and close this.
